packages/components/src/internal/components/user/UserDetailsPanel.tsx

```diff
@@ -84,12 +84,14 @@ export class UserDetailsPanel extends React.PureComponent<Props, State> {
                     this.setState(() => ({ userProperties: undefined, loading: false }));
                 });
 
-            try {
-                const groups = await getUserGroups(userId);
-                this.setState(() => ({ groups }));
-            } catch (e) {
-                console.error(e);
-            }
+            getUserGroups(userId)
```

Comment:
was there a specific reason to move from the try/catch to a then()?

Reply:
Mainly just to align with the form used in getUserProperties above—fairly arbitrary but looks a bit nicer.